sparkplug/timereporters/logger.py

```diff
@@ -0,0 +1,40 @@
+"""
+Sends timing information to the logs
+"""
+
+import logging
+from sparkplug.logutils import LazyLogger
+from sparkplug.timereporters.base import Base, mn_md_mx
+
+_log = LazyLogger(__name__)
+
+class Logger(Base):
+    def __init__(self, aggregation_count, level="DEBUG"):
+        super().__init__(aggregation_count)
+        self.exec = []
+        self.erro = []
+        self.wait = []
+        # get the exact method for the correct logging level:
+        self.logger = getattr(_log, str(level).lower())
+        assert(callable(self.logger), "Logging level on timereporters.Logger does not resolve to anything useful")
+
+    def append_exec(self, delta):
+        self.exec.append(delta)
+        if len(self.exec) >= self.aggregation_count :
+            mn, md, mx = mn_md_mx(self.exec)
```

Comment:
I am not sure if these are conventions or not, but I had to do a search to figure out what `mn, md, mx` meant.
Could you consider a rename?

Reply:
Done.